include/rocksdb/options.h

```diff
@@ -393,18 +393,14 @@ struct DBOptions {
   // Default: true
   bool paranoid_checks = true;
 
-  // If true, track WALs in MANIFEST and verify them on recovery.
+  // If true, the synced WAL's log number and size are tracked in MANIFEST,
```

Comment:
Since there can be multiple sync'ed WALs, so should we say "the log numbers and sizes of synced WALs"?